### PR TITLE
Removed unused code, added some translations

### PR DESCRIPTION
**Pull request by uweseimet**
Most of the comments in filepath.cpp are actually useless, as they don't provide any information. e.g.

//---------------------------------------------------------------------------
//
//      Clear
//
//---------------------------------------------------------------------------
void FASTCALL Filepath::Clear()

I wonder if removing these redundant comments makes the code more readable. Any thoughts on this?

### CODE REVIEW

File: src/raspberrypi/filepath.cpp [hunk 1]
```diff
@@ -79,10 +79,10 @@ void FASTCALL Filepath::SetPath(LPCSTR path)
 	ASSERT(path);
 	ASSERT(strlen(path) < _MAX_PATH);
 
-	// パス名コピー
+	// Copy pathname
 	strcpy(m_szPath, (LPTSTR)path);
 
-	// 分離
+	// Solit
```
Comment (akuker) [inline src/raspberrypi/filepath.cpp:85]: Should this be "Split"?

File: src/raspberrypi/filepath.cpp [hunk 2]
```diff
@@ -206,55 +206,6 @@ void FASTCALL Filepath::Split()
...
-	// strlenで調べたとき、最大59になるように細工
-	ShortName[59] = '\0';
-
-	// const charとして返す
-	return (const char*)ShortName;
-}
-
 //---------------------------------------------------------------------------
 //
 //	ファイル名＋拡張子取得
```
Comment (akuker) [inline src/raspberrypi/filepath.cpp:211]: Translation of this is:

"File name + extension acquisition"
"The returned pointer is temporary. Copy immediately"

**Comment by akuker** [comment]
Some minor changes requested. Thank you!!

File: src/raspberrypi/filepath.cpp (hunk 1 above)
Comment (uweseimet) [reply src/raspberrypi/filepath.cpp]: Indeed, I just fixed it. But this just shows how useless certain comments are when one line below the method of the same name is called.

Comment (akuker) [reply src/raspberrypi/filepath.cpp]: I absolutely agree! In hindsight, I could have just left all of those comments in Japanese (or deleted them). There were only a couple of them that were truly useful.

**Review by akuker — APPROVED** [review]

File: src/raspberrypi/filepath.cpp (hunk 1 above)
Comment (uweseimet) [reply src/raspberrypi/filepath.cpp]: The problem with comments that do not provide any relevant piece of information is that they make it harder to read the code, and you see less code on one screen, which makes it harder to assess the full context.

**Pull request merged**